lib/web_ui/lib/src/engine/canvaskit/surface.dart

```diff
@@ -26,14 +26,14 @@ typedef SubmitCallback = bool Function(SurfaceFrame, CkCanvas);
 
 /// A frame which contains a canvas to be drawn into.
 class SurfaceFrame {
-  final CkSurface skiaSurface;
-  final SubmitCallback submitCallback;
-  bool _submitted;
-
   SurfaceFrame(this.skiaSurface, this.submitCallback)
       : _submitted = false,
         assert(skiaSurface != null), // ignore: unnecessary_null_comparison
-        assert(submitCallback != null); // ignore: unnecessary_null_comparison
+        assert(submitCallback != null);
+
+  final CkSurface skiaSurface;
+  final SubmitCallback submitCallback;
+  final bool _submitted; // ignore: unnecessary_null_comparison
```

Comment:
What is this `// ignore: unnecessary_null_comparison` referring to?

Reply:
Looks like `dart fix` messed up the comments. In general, `unnecessary_null_comparison` is ignored on a global level, so individual ignores are no longer needed. I will remove them here.